src/eos-kolibri-manage.in

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -e
+
+if [ "$EUID" -ne "0" ]; then
+  echo "$0 is not running as root. Try using sudo."
+  exit 2
+fi
+
+: ${KOLIBRI_HOME:="@KOLIBRI_DATA_DIR@"}
+
+if [ ! -f "${KOLIBRI_HOME}/db.sqlite3" ]; then
+  echo "There is no Kolibri database in '${KOLIBRI_HOME}'."
+  echo "Try running Kolibri before running this command."
+  exit 1
+fi
+
+# To use flatpak-run with `sudo -u`, we need to unset SUDO_COMMAND:
+# <https://github.com/flatpak/flatpak/blob/1.10.2/common/flatpak-run.c#L3642-L3661>
+
+sudo -u @KOLIBRI_USER@ env -u SUDO_COMMAND - flatpak run \
```

Comment:
You might also try `runuser -u @KOLIBRI_USER@ -- flatpak run ...`. `runuser` only works from root but I find it to be cleaner in that scenario.

Reply:
Ah, that's nicer indeed. Thanks :)